src/main/groovy/net/vivin/gradle/versioning/tasks/TagTask.groovy

```diff
@@ -13,11 +14,24 @@ import org.gradle.tooling.BuildException
  * @author vivin
  */
 class TagTask extends DefaultTask {
+
+    Closure<String> tagMessage = {
```

Comment:
As it is inside tag task, i wouldn't call it `tagMessage` but simply `message`.
I also added a `message` method to set the property.
So now you can write the much nicer version `tag { message fromProjectProperty('tagMessage') }`, compared to `tag { tagMessage = fromProjectProperty('tagMessage') }`.